R/grade_this.R

```diff
@@ -142,6 +142,11 @@ grade_this <- function(
       check_env <- list2env(check_env, envir = new.env(parent = emptyenv()))
     }
 
+    empty_code <- empty_code(check_env[[".user_code"]])
+    if (!is.null(empty_code)) {
+      return(empty_code)
+    }
+
```

Comment:
Maybe `grade_this()` shouldn't be strict about empty code? We don't _know_ that we're going to need the user code yet ... whereas `grade_this_code()` and `fail_if_code_feedback()` require code _a priori_.

Reply:
That's a good point. Changed in [6bd364a](https://github.com/rstudio/gradethis/pull/288/commits/6bd364a60343fd898195bffc0de645e04f526db9)